arch/ARM/ARMMappingInsn.inc

```diff
@@ -12922,25 +12922,25 @@
 {
 	ARM_tBL, ARM_INS_BL,
 #ifndef CAPSTONE_DIET
-	{ ARM_REG_PC, 0 }, { ARM_REG_LR, 0 }, { ARM_GRP_THUMB, ARM_GRP_CALL, 0 }, 1, 0
+	{ ARM_REG_PC, 0 }, { ARM_REG_LR, 0 }, { ARM_GRP_THUMB, 0 }, 1, 0
```

Comment:
BL & BLX are actually CALL instructions (in the sense that they save the returned address for later return), so please keep ARM_GRP_CALL. at the same time, please add this group to other part of this PR.

thanks for your effort!